x-pack/plugins/secops/server/graphql/who_am_i/schema.gql.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import gql from 'graphql-tag';
+
+export const whoAmISchema = gql`
+  type SayMyName {
+    "The id of the source"
+    appName: String!
+  }
+
+  extend type Source {
+    whoAmI: SayMyName
```

Comment:
nit: I'd add the description even if this is throw away?

Does make me wonder if we have a `linter` to fail on this or other schema must-have's we add :-)

I'll add a discussion item to the backlog about this pre-commit hook even though I'm skiddish about pre-commit hooks:
https://github.com/cjoudrey/graphql-schema-linter